src/lib.rs

```diff
@@ -412,4 +451,31 @@ mod tests {
         assert_eq!(person.downcast_mut::<Chris>(), None);
         assert!(person.downcast::<Chris>().err().is_some());
     }
+
+    #[test]
+    fn parameterised() {
```

Comment:
Fixed. Started writing everything as "parameterised" but later decided on the US spelling and missed converting that instance.

On a side note, as a non-native English speaker these spelling differences are quite annoying: I have learnt or heard both I often have to look up which one to use :/.